packages/synapse-react-client/src/components/Icon/Icon.tsx

```diff
@@ -98,6 +98,7 @@ export type IconProps = {
 const iconSvgMapping = {
   clipboard: <IconSvg icon="clipboard" />,
   [DATASET]: <IconSvg icon="dataset" />,
+  [EXTERNAL_RESOURCE]: <IconSvg icon="link" />,
```

Comment:
Adam also requested we change the icon for the External Resources:

<img width="1382" alt="Screen Shot 2024-03-14 at 2 45 58 PM" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/1864447/efcb7f5b-5f81-4a13-b091-f931f3d0b79b">